lib/node_modules/@stdlib/repl/README.md

```diff
@@ -176,6 +198,167 @@ repl.clearUserDocs();
 repl.close();
 ```
 
+#### REPL.prototype.themes()
+
+Returns a list of all available themes for syntax highlighting.
+
+```javascript
+var debug = require( '@stdlib/streams/node/debug-sink' );
+
+// Create a new REPL:
+var repl = new REPL({
+    'output': debug()
+});
+
+// ...
+
+// Fetch all available themes:
+var themes = repl.themes();
+// returns [...]
+
+// ...
+
+// Close the REPL:
+repl.close();
+```
+
+#### REPL.prototype.getTheme()
```

Comment:
@Snehil-Shah The method signatures are still missing parameters for these methods.